### PR TITLE
add new katalyst environment

### PR DESCRIPTION
**Pull request by dtwilliamson**
katalyst01 is a special server without the usual Openstax setup for Katalyst to have their way with. 

Other changes may be needed because of the unusual configuration. The changes in that commit, if it comes, should probably not be merged, but this PR doesn't affect any code.

### CODE REVIEW

File: environments/devb/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+You will also need the vault password to decrypt the file at
+``environments/cte-dev/group_vars/all/vault.yml``.
+
+## Usage
+
+For provisioning the cnx-suite onto a VM:
+
+```sh
+ansible-playbook -i environments/dev/inventory main.yml --ask-become-pass --ask-vault-pass
```
[the remark this answers is not in the document]
Comment (dtwilliamson) [reply environments/devb/README.md]: Yeah, I ignored the README. #WILLFIX

[the remark this answers is not in the document]
Comment (dtwilliamson) [reply environments/devb/README.md]: I also forgot to rename the environment in the path

**Comment by karenc** [comment]
Is the second commit meant to be a `mv`?  We now have `environments/devb` and `environments/katalyst01`

**Comment by reedstrm** [comment]
@karenc  - no @helenemccarron requested a second dev environment primarily for the textbook devs, since dev tends to get reset/restarted/updated somewhat randomly (mostly by me :-) )

**Comment by karenc** [comment]
:o We're adding `environments/devb` and `environments/katalyst01` in this PR and they're both using `devb.cnx.org`.

**Comment by dtwilliamson** [comment]
I've apparently gotten git into a twist. There are supposed to be two separate PRs - one for devb and one for katalyst01. I would appreciate help in getting them untwisted.

**Comment by karenc** [comment]
Oh... right.  let me separate them.  I'll have the `environments/katalyst01` in this PR and `environments/devb` in another.

**Pull request merged**